src/components/NetworkSwitcher.tsx

```diff
@@ -0,0 +1,761 @@
+                    gap: 12,
+                    alignItems: 'center',
+                    marginTop: 68 - height,
+                  }}
+                >
+                  <LinearGradient
+                    colors={['#268FFF1F', '#268FFF14']}
+                    angle={135}
+                    useAngle
+                    style={{
+                      height: 36,
+                      width: 36,
+                      borderRadius: 10,
+                      borderWidth: 1,
+                      borderColor: '#268FFF0D',
+                      backgroundColor: '#268FFF14',
+                      alignItems: 'center',
+                      justifyContent: 'center',
+                    }}
+                  >
+                    <Text weight="heavy" size="17pt" color={{ custom: blue }}>
+                      􀍱
+                    </Text>
+                  </LinearGradient>
+                  <View style={{ gap: 10 }}>
+                    <Text weight="heavy" size="15pt" color="labelSecondary">
+                      {i18n.t(t.customize_networks_banner.title)}
+                    </Text>
+                    <Text weight="semibold" size="13pt" color="labelQuaternary">
+                      {/* 
+                    is there a way to render a diferent component mid sentence?
+                    like i18n.t(t.customize_networks_banner.description, { Edit: <Text... /> })
+                  */}
+                      Tap the{' '}
+                      <Text weight="bold" size="13pt" color={{ custom: blue }}>
+                        Edit
+                      </Text>{' '}
+                      button below to set up
+                    </Text>
+                  </View>
+                  <Pressable style={{ marginLeft: 'auto', height: '100%' }} onPress={dismissCustomizeNetworksBanner}>
+                    <Text weight="heavy" size="13pt" color="labelQuaternary">
+                      􀆄
+                    </Text>
+                  </Pressable>
+                </View>
+              </AnimatedBlurView>
+            </MaskedView>
+          </Animated.View>
+        </DesignSystemProvider>
+      );
+    };
+
+const useNetworkOptionStyle = (isSelected: SharedValue<boolean>, color: string) => {
+  const { isDarkMode } = useColorMode();
+
+  const surfacePrimary = useBackgroundColor('surfacePrimary');
+  const networkSwitcherBackgroundColor = isDarkMode ? '#191A1C' : surfacePrimary;
+
+  const defaultStyle = {
+    backgroundColor: isDarkMode ? globalColors.white10 : globalColors.grey20,
+    borderColor: '#F5F8FF05',
+  };
+  const selectedStyle = {
+    backgroundColor: chroma.scale([networkSwitcherBackgroundColor, color])(0.16).hex(),
+    borderColor: chroma(color).alpha(0.16).hex(),
+  };
+
+  const scale = useSharedValue(1);
+  useAnimatedReaction(
+    () => isSelected.value,
+    () => {
+      scale.value = withSequence(withTiming(0.95, { duration: 50 }), withTiming(1, { duration: 80 }));
+    }
+  );
```

Comment:
this is causing the deselected network to run the same animation as the selected network, should only be if isSelected.value is true